data/team/members.yaml

```diff
@@ -106,52 +141,105 @@ members:
     - icon: github
       text: GitHub
       url: https://github.com/yonghui-cc
-    - icon: info-circle
+    - icon: book
       text: ORCiD
       url: https://orcid.org/0000-0002-8201-2894
-- name: David Schaller
-  role: Postdoc
-  project: Free energy calculations and machine learning for kinase drug discovery
-  image: /images/team/davidschaller.jpg
+- name: Lisa Chiara Gosch
+  prefix:
+  role: Dr. med. candidate
+  status: member
+  key: chiara.gosch
+  project: Computational and experimental testing of novel EGFR and HDAC inhibitors (co-supervised with Prof. Höpfner)
   links:
     - icon: envelope
-      text: david.schaller@charite.de
-      url: mailto:david.schaller@charite.de
+      text: lisa-chiara.gosch@charite.de
+      url: mailto:lisa-chiara.gosch@charite.de
     - icon: linkedin
-      text: LinkedIn
-      url: https://www.linkedin.com/in/david-schaller-2a68b2133/
+      text: Linkedin
+      url: https://www.linkedin.com/in/lisa-chiara-gosch-80022070/
+# --MASTERS--
+- name: Michele Wichmann
+  prefix:
+  key: michele.wichmann
+  role: Master student
+  status: studentmember
+  image:
+  project: Dynamic pharmacophore modeling from apo structures
+- name: Allen Dumler
+  prefix:
+  key: allen.dumler
+  role: Bachelor student
+  status: studentmember
+  image:
+  project: Standardization pipelines in computer-aided drug design
+
+  # --ALUMNI--
+- name: Paula Schmiel
+  prefix:
+  key: paula.schmiel
+  role: Master student and short term scientist
+  status: alumni
+  image: /images/team/paulaschmiel.jpg
+  project: Subpocket-based fragmentation and recombination of kinase inhibitors
+  links:
     - icon: github
       text: GitHub
-      url: https://github.com/schallerdavid
-    - icon: info-circle
-      text: ORCiD
-      url: https://orcid.org/0000-0002-1881-4518
-alumni:
-- name: Lisa Chiara Gosch
-  role: PhD candidate
-  project:
-  date:
+      url: https://github.com/paulaju
+- name: Robert Strothmann
+  prefix:
+  role: Bachelor student
+  status: alumni
+  key: robert.strothmann
+  project: Automatic generation of T²F-Flex pharmacophores based on MD simulations
 - name: Henry Webel
+  prefix:
+  key: henry.webel
+  status: alumni
   role: Scientist
   project: Predicting cytotoxicity using deep neural networks
-  date:
-- name: Dr. Jérémie Mortier
+- name: Boran Adas
+  prefix:
+  key: boran.adas
+  status: alumni
+  role: Master thesis and GoogleSummerOfCode
+  project: Robust machine learning pipeline for classification problems in cheminformatics
+- name: Jérémie Mortier
+  prefix: Dr.
+  key: jeremie.mortier
+  status: alumni
   role: Scientist
```

Comment:
did you on purpose write "role: Scientist" and not "role: Postdoc"?

Reply:
@AndreaVolkamer, I need your feedback here :)

Reply:
Every PostDoc is a scientist :) (I think I used it because he was only so short time here and also already a senior PostDoc) I would just keep it.